api/middleware/applicationValidation.js

```diff
@@ -0,0 +1,12 @@
+const validation = (schema) => async (req, res, next) => {
+  const body = req.body;
+
+  try {
+    await schema.validate(body);
+    next();
+  } catch (err) {
+    return res.status(400).json({ message: err.message });
```

Comment:
This should just be calling `next({ status: 400, message: err.message })`. We already have global error handling middleware set up.

Reply:
Changed!